packages/web-runtime/src/container/bootstrap.ts

```diff
@@ -676,6 +681,51 @@ export const announceVersions = ({
   })
 }
 
+/**
+ * announce updates
+ *
+ * @param updateStore
+ * @param capabilityStore
+ * @param configStore
+ * @param clientService
+ */
+export const announceUpdates = async ({
+  updatesStore,
+  capabilityStore,
+  configStore,
+  clientService
+}: {
+  updatesStore: UpdatesStore
+  capabilityStore: CapabilityStore
+  configStore: ConfigStore
+  clientService: ClientService
+}): Promise<void> => {
+  if (!capabilityStore.capabilities.core['check-for-updates']) {
+    return
+  }
+
+  try {
+    updatesStore.setIsLoading(true)
```

Comment:
Why do you set loading to `true` here? It's initialized with `true`.

Reply:
just in case, if we re-use this function, e.G polling or something else, this would come in handy